### PR TITLE
Feature table functionality

### PR DESCRIPTION
**Pull request by omokshyna**
Added manual integration of features and general description of the feature table

### CODE REVIEW

**Comment by SteffenHeu** [comment]
do you know why the actions are not running?

**Comment by omokshyna** [comment]
@SteffenHeu I think the reason is that pull-request is not included in the actions triggers. Actions run in the fork and after PR is merged.

File: mkdocs.yml [hunk 1]
```diff
@@ -141,12 +142,12 @@ plugins:
...
-      author: The MZmine Community
-      cover_title: MZmine 3
-      cover_subtitle: Documentation
-      cover_logo: img/MZmine_logo_CMYK.png
+#  - with-pdf:
+#      output_path: documentation.pdf
+#      author: The MZmine Community
+#      cover_title: MZmine 3
+#      cover_subtitle: Documentation
+#      cover_logo: img/MZmine_logo_CMYK.png
```
Comment (SteffenHeu) [inline mkdocs.yml:150]: why?

**Comment by SteffenHeu** [comment]
I'm confused, PRs to the master branch are included in the workflows.

File: mkdocs.yml (hunk 1 above)
Comment (omokshyna) [reply mkdocs.yml:150]: Fixing it :) I commented it to use mkdocs locally.

**Comment by omokshyna** [comment]
You're right, they are. That's weird, the actions have been running on all the latest PRs as well. I'll monitor this.

**Pull request merged**